src/master-playlist-controller.js

```diff
@@ -478,6 +517,27 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.tech_.trigger({type: 'usage', name: 'hls-playlist-cue-tags'});
     }
   }
+
+  shouldSwitchToMedia_(nextPlaylist) {
+    const currentPlaylist = this.masterPlaylistLoader_.media();
```

Comment:
We were calling this function the same way everywhere except for `nextPlaylist` so I just added it to the mpc object.